forwards_paper.tex

```diff
@@ -293,45 +295,31 @@ \subsection*{Simulation benchmarks}
 
 
 %%%%%%
-\subsection*{Working with a tree sequence through tables}
+\subsection*{Tables for succinct tree sequences}
 
-The simulations above recorded information about each new individual
-in a collection of tables that together define a \emph{tree sequence}.
+We now explain what we actually did to achieve a $50\times$ speedup.
+The ``pedigree recording'' simulations above recorded information about each new individual
+in a collection of tables that together define a \emph{succinct tree sequence}
+(or, simply ``tree sequence'').
 A {tree sequence} is an encoding for a sequence of correlated trees,
 such as those describing the history of a sexual population.
-A closely related structure is known as the \emph{ancestral recombination graph} --
-the {ARG} \citep{griffiths1991two,griffiths1997ancestral},
-where nodes are common ancestor or recombination events --
-which has been the subject of substantial study
-under the assumptions of coalescent
-theory~\citep{wiuf1997number,wiuf1999ancestry,marjoram2006coalescent,wilton2015smc}.
-However, the properties of the ARG as a computational structure have not
-been studied and, despite several efforts to standardise a common
-format~\citep{morin2006netgen,mcgill2013graphml}, % TODO check these refs against others in msprime paper.
-ARGs are rarely used
-in practise. In contrast, the algorithmic properties of tree sequence
-algorithms have been explored in detail~\citep{kelleher2016efficient},
-contributing substantially to the efficiency of the \msprime\ coalescent simulator.
-
 Tree sequences are efficient because branches that are shared by adjacent trees are stored once,
 rather than repeatedly for each tree.
 The topology of a tree sequence is defined via its \emph{nodes} and \emph{edges},
-while information about variants are recorded as \emph{sites} and \emph{mutations}.
-The format for these four tables is given in more detail in the Methods;
-here we explain conceptually,
-using the example of Figure~\ref{fig:example_tree_sequence}. This formulation
-is derived from the ``coalescence records'' encoding of tree
+while information about variants is recorded as \emph{sites} and \emph{mutations};
+we give an example in Figure~\ref{fig:example_tree_sequence}. 
+This formulation is derived from the ``coalescence records'' encoding of tree
 sequences~\citep{kelleher2016efficient}, normalised to remove redundancy
-and generalised to include a much larger class of tree topology.
+and generalised to include a more general class of tree topologies.
 
 The \emph{nodes} of a tree sequence
-correspond to the vertices in the individual genealogies along the sequence,
-and each node refers to a distinct ancestor.
-Since each node represents a specific ancestor, it has a unique ``time'',
-thought of as her birth time, which determines the height of any vertices
```

Comment:
nodes are haploid, so don't have a sex (and I think it reads better with 'the node')

Reply:
agreed